README.md

```diff
@@ -77,6 +77,23 @@ The first time you sign in with any of the users, you'll have to enter their tem
 
 Feel free to play around with this demo, along with the Cognito config, to understand better how it works!
 
+## Using the TOTP MFA
+
+The library offers English and French translations for TOTP MFA pages. If you need other translations, have a look to the [ra-auth-cognito-language-french package](packages/ra-auth-cognito-language-french/README.md).
```

Comment:
Please provide an absolute link to `packages/ra-auth-cognito-language-french/README.md` otherwise it won't work when we publish on npm